apps/frontend/src/libs/enums/error-message.enum.ts

```diff
@@ -0,0 +1,5 @@
+const ErrorMessage = {
+	UNEXPECTED_ERROR: "An unexpected error occurred.",
```

Comment:
Let's have it as a const and set as a default value for methods in notification module